Portfolio/src/assets/App.css

```diff
@@ -0,0 +1,119 @@
+html,
+body {
+  background-color: #d4abb2;
+  font-family: 'Raleway';
+}
+
+.menuButton {
+  display: flex;
+  flex-direction: column;
+  background-color: transparent;
+  border: none;
+  position: fixed;
+  top: 20px;
+  right: 20px;
+  z-index: 1000;
+}
+
+.router-wrapper {
+  font-family: 'Raleway';
+  text-transform: uppercase;
+}
+
+.navbar {
+  display: flex;
+  justify-content: flex-end;
+  font-size: 25px;
+  gap: 60px;
+  list-style: none;
+}
+
+.navbar li a {
```

Comment:
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css

Read this very nice article above. I can recommend to stop targeting html tags in CSS but use classnames instead :)

It is important for u to understand the basics, so do less code and more reading when we share resources. There's a long way to go for you.